tests/attestation_plugin/plugin/tests.c

```diff
@@ -105,6 +105,191 @@ static void _test_and_unregister_verifier()
     OE_TEST(oe_unregister_verifier(&mock_verifier2) == OE_NOT_FOUND);
 }
 
+static bool _find_format_id(
```

Comment:
Please consider update the attestation_plugin tests so that you can run both v2 and v3 configurations (i.e. update the makefile so it builds both variants) so that they will get tested through CI/CD until the older variant is removed, otherwise, it's likely CI/CD will not catch regressions in the v2 API behavior as it only runs with the default build configuration. The v2 configuration tests should also continue to test the old semantics (i.e. preserve the old tests so you know you have parity testing)